lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf

```diff
@@ -0,0 +1,471 @@
+            zh: "Kafka 消息模版"
+        }
+    }
+    kafka_message_key {
+        desc {
+            en: "Template to render Kafka message key. "
+                "If the desired variable for this template is not found in the input data "
+                "<code>NULL</code> is used"
+            zh: "生成 Kafka 消息 Key 的模版。当所需要的输入没有时，会使用 <code>NULL</code>。"
+        }
+        label {
+            en: "Message Key"
+            zh: "消息的 Key"
+        }
+    }
+    kafka_message_value {
+        desc {
+            en: "Template to render Kafka message value."
+                "If the desired variable for this template is not found in the input data "
+                "<code>NULL</code> is used"
+            zh: "生成 Kafka 消息 Value 的模版。当所需要的输入没有时，会使用 <code>NULL</code>。"
+        }
+        label {
+            en: "Message Value"
+            zh: "消息的 Value"
+        }
+    }
+    kafka_message_timestamp {
+        desc {
+            en: "Which timestamp to use. "
+                "The timestamp is expected to be a millisecond precision Unix epoch "
+                "which can be in string format, e.g. <code>1661326462115</code> or "
+                "<code>'1661326462115'</code>."
+                "When the desired data field for this template is not found, "
+                "or if the found data is not a valid integer, "
+                "the current system timestamp will be used."
+            zh: "生成 Kafka 消息时间戳的模版。"
+                "该时间必需是一个整型数值（可以是字符串格式）例如 <code>1661326462115</code> "
+                "或 <code>'1661326462115'</code>。"
+                "当所需的输入字段不存在，或不是一个整型时，"
+                "则会使用当前系统时间。"
+        }
+        label {
+            en: "Message Timesstamp"
+            zh: "消息的时间戳"
+        }
+    }
+    max_batch_bytes {
+        desc {
+            en: """Maximum bytes to collect in a Kafka message batch.
+Most of the Kafka brokers default to a limit of 1MB batch size.
+EMQX's default The default value is less than 1MB in order to compensate
+Kafka message encoding overheads (especially when each individual message is very small).
+When a single message is over the limit, it is still sent (as a single element batch)."""
+            zh: """最大消息批量字节数。
+大多数 Kafka 环境的默认最低值是 1MB，EMQX 的默认值比 1MB 更小是因为需要
+补偿 Kafka 消息编码索需要的额外字节（尤其是当每条消息都很小的情况下）。
+当单个消息的大小超过该限制时，它仍然会被发送，（相当于该批量中只有单个消息）。"""
+        }
+        label {
+            en: "Max Batch Bytes"
+            zh: "最大批量字节数"
+        }
+    }
+    compression {
+        desc {
+            en: "Compression method."
+            zh: "压缩方法。"
+        }
+        label {
+            en: "Compression"
+            zh: "压缩"
+        }
+    }
+    partition_strategy {
+        desc {
+            en: "Partition strategy is to tell the producer how to dispatch messages to Kafka partitions.\n\n"
+                "<code>random</code>: Randomly pick a partition for each message\n"
+                "<code>key_dispatch</code>: Hash Kafka message key to a partition number\n"
```

Comment:
Missing `roundrobin`?

Reply:
no plan to supporting it in 5.0.

Reply:
> no plan to supporting it in 5.0.

because it makes use of the process dictionary, when multiple callers running concurrently, it's essentially random.

Reply:
But it is allowed by the schema:

https://github.com/emqx/emqx/pull/8798/files#diff-917c1719cc75abe4e935e1a1fa77fef1665b4dcf02f5802de573292fdb72dff6R152